local/dubug_environment.md

```diff
@@ -0,0 +1,64 @@
+# Debug environment for 0Chain go executables
```

Comment:
This doc is really valuable, I'll look at it closer little bit later.

Reply:
I need to redo the screen prints, plus add a start debugging checklist.